src/core/Akka.Cluster.Tests/Serialization/ClusterMessageSerializerSpec.cs

```diff
@@ -63,20 +67,6 @@ public void Can_serialize_HeartbeatRsp()
             AssertEqual(message);
```

Comment:
This test only succeeds because `-1,`, -`1` are the hard-coded values when legacy==on. Need to harden this to make sure that the correct, non-`-1` values are supported when `legacy==off`.

Reply:
The `AssertEqual` regardless of config is what's not going to work here. Change 

```csharp
 var message = new ClusterHeartbeatSender.HeartbeatRsp(uniqueAddress, -1, -1);
```

to 

```csharp
 var message = new ClusterHeartbeatSender.HeartbeatRsp(uniqueAddress, 10, 3);
```

When legacy off, 10,3

When legacy on, -1, -1

Reply:
Done